src/howdryiam/DRYWithASideOfLoops.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package howdryiam;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ *
+ * @author Rhonda Marshall
+ * Oct 4, 2015
+ */
+public class DRYWithASideOfLoops {
+
+    /**
+     * Driver to show how this problem could be encapsulated in a loop
+     */
+    public static void main() {
+        Scanner keyboard = new Scanner(System.in);
+        
+        // An ArrayList to store the LabelPrinter instances
+        ArrayList<LabelPrinter> labelPrinterList = new ArrayList<>();
```

Comment:
I originally had ArrayList<LabelPrinter> labelPrinterList = new ArrayList<LabelPrinter>();
NetBeans kept telling me "redundant type arguments in new expression (use diamond operator instead). 
 I googled this and decided to remove LabelPrinter  from new expression. As suspicious as I was about this, it did stop NetBeans from complaining.
Was I wrong to do this?